src/pretalx/orga/templates/orga/cfp/question_view.html

```diff
@@ -45,13 +46,21 @@
                         {{ question.get_target_display }}
                     </td>
                     <td>
-                        <i class="fa fa-{{ question.required|yesno:'check-circle text-success,times-circle text-danger' }}">
-
-                        </i>
+                        {% if question.question_required == "require" %}
+                            {% translate "always" %}
+                        {% elif question.question_required == "require after"%}
+                            {% translate "after" %}
+                        {% elif question.question_required == "freeze after"%}
+                            {% translate "before" %}
+                        {% else %}
+                            {% translate "never" %}
+                        {% endif %}
+                        {% if question.deadline %}
```

Comment:
Alternative suggestion: Let's keep the "required" column to a yes/no indicator that reflects if the question is currently required (using the `required` property I suggested earlier). Then add another indicator, a clock or the like, to signal if this property is based on a deadline or not.

Otherwise, this part of the table will start being unreasonably wide. We could put the deadline in the alt text for the clock icon.

Reply:
- [x] ```required``` table column like before changes
- [X] ```deadline``` table column

Reply:
![Στιγμιότυπο οθόνης (315)](https://user-images.githubusercontent.com/56487775/123092385-16732e80-d433-11eb-9716-7847db9ed2b1.png)

Reply:
![Στιγμιότυπο οθόνης (316)](https://user-images.githubusercontent.com/56487775/123092767-841f5a80-d433-11eb-8ee1-0cf95f8c01e6.png)